src/python/pants/base/deprecated.py

```diff
@@ -72,11 +72,11 @@ def validate_deprecation_semver(version_string: str, version_description: str) -
     :raises DeprecationError: if the version_string parameter is invalid.
     """
     if version_string is None:
-        raise MissingSemanticVersionError(f"The {version_string} must be provided.")
+        raise MissingSemanticVersionError(f"The {version_description} must be provided.")
```

Comment:
Why is this? It does not correspond with the iF statement

Reply:
Eh.. `"The None must be provided"` seemed like a mistake to me. This change gives `"The removal version must be provided"` (or `"The deprecation start version must be provided"`).

Reply:
Oh, OK that makes more sense